examples/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -25,8 +25,35 @@ namespace ArborX
 namespace Details
 {
 
+// Check that core points have nonnegative indices
+template <typename ExecutionSpace, typename IndicesView, typename OffsetView,
+          typename LabelsView>
+bool verifyCorePointsNonnegativeIndex(ExecutionSpace const &exec_space,
+                                      IndicesView /*indices*/,
+                                      OffsetView offset, LabelsView labels,
+                                      int core_min_size)
+{
+  int n = labels.size();
+
+  int num_incorrect = 0;
+  Kokkos::parallel_reduce(
+      "ArborX::DBSCAN::verify_core_points_nonnegative",
+      Kokkos::RangePolicy<ExecutionSpace>(exec_space, 0, n),
+      KOKKOS_LAMBDA(int i, int &update) {
+        bool self_is_core_point = (offset(i + 1) - offset(i) >= core_min_size);
+        if (self_is_core_point && labels(i) < 0)
```

Comment:
only possible negative value is -1 to denote noise right?

Reply:
Correct.